Documentation/kubernetes-upgrade.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading Kubernetes
+
+This document will describe upgrading your cluster's master and worker nodes. For general information on Kubernetes cluster management, upgrades (including more advanced topics such as major API version upgrades) see the [Kubernetes upstream documentation](http://kubernetes.io/v1.0/docs/admin/cluster-management.html).
```

Comment:
This document describes upgrading the Kubernetes components on a cluster's master and worker nodes.